api-tests/README.md

```diff
@@ -16,42 +16,41 @@
 
 ## Running the tests
 
-Run using the main `.env` file by using `npm test`.
-
-Additional arguments can be passed to cucumber, e.g. to include specific tags: `npm test -- --tags '@Build'`
-
-You can also run against locally by using the presets:
+There are 3 presets that can be used:
 
+- `npm run test:build` - runs against the deployed build environment
 - `npm run test:local` - runs against an already-running local instance of core-back
 - `npm run test:ci` - starts a new local instance of core-back and tests against it
```

Comment:
(this will require people to change their config, sorry :( )

Reply:
Alternatively we could ask people to set the env var on their system, but I always find that a bit unsatisfactory